tests/integrationv2/test_pq_handshake.py

```diff
@@ -154,8 +184,8 @@ def test_s2nc_to_s2nd_pq_handshake(managed_process, protocol, client_cipher, ser
         port=port,
         protocol=protocol,
         cipher=server_cipher,
-        cert=Certificates.RSA_4096_SHA512.cert,
```

Comment:
Does this need to be changed? Seems like the certificate parameter is still RSA_4096_SHA512.

Reply:
It doesn't need to be. I changed this test to now pass this cert into invalid_test_parameters here: https://github.com/aws/s2n-tls/blob/690c3e32c0139bdfa99dd636ad4bed10a8a5d826/tests/integrationv2/test_pq_handshake.py#L150

I changed server_options to use the certificate argument so the cert isn't hardcoded twice, in case it's updated or more are added. Though, if it makes more sense and should be more clear that this one specific cert is being tested, I can revert this.

Reply:
nvm this is fine.